api/src/integrations/epic/datasource.js

```diff
@@ -152,14 +152,26 @@ class DataSource {
             continue;
           }
 
-          // Check if Order/Inspection
+          // Check if Inspection or Order and that it is not a Fee Order.
           const rec = await recordTypeUtils.transformRecord(theRecord);
-          if ((rec._schemaName === "Order") || (rec._schemaName === "Inspection")) {
-            processRecords.push(theRecord);
+          if (rec._schemaName === 'Inspection' || rec._schemaName === 'Order') {
+              if (!recordTypeUtils.isRecordFeeOrder(rec)) {
+                processRecords.push(theRecord);
+              }
           }
           // Skip everything else
         } else {
-          processRecords.push(theRecord);
+          const rec = await recordTypeUtils.transformRecord(theRecord);
+
+          // Check if Order or Inspection and not a Fee Order 
+          if (rec._schemaName === 'Inspection' || rec._schemaName === 'Order') {
```

Comment:
You have the condition:
`if (rec._schemaName === 'Inspection' || rec._schemaName === 'Order')`
Before calls to `recordTypeUtils.isRecordFeeOrder()`

Would it make more sense to push the condition into isRecordFeeOrder so you can remove the nested "Ifs"

Reply:
It actually does exist in the `isRecordFeeOrder` function as well. However, many different record types come through here, but that function only exists on the two that need it. So if something other than Inspection or Order where to arrive, they can not call 'isRecordFeeOrder' as it won't exist.

Reply:
Removed the check by moving where the `isRecordFeeOrder` function lives.